src/components/common/modal/WithdrawalModal.tsx

```
@@ -0,0 +1,79 @@
+
+  return (
+    <Modal
+      transparent
+      animationType="slide"
+      visible={visible}
+      onRequestClose={onCancel}
+    >
+      <S.Backdrop>
+        <S.ModalContainer>
+          <S.TextBox padding="32px 0px 16px">
+            <Typography font="bold20" color="defaultBlack" children={title} />
```

Biome (2.4.6)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)